worker/src/main.rs

```diff
@@ -672,3 +672,11 @@ pub enum WorkerRequest {
 pub enum WorkerResponse<V: Encode + Decode> {
     ChainStorage(Vec<u8>, Option<V>, Option<Vec<Vec<u8>>>), // (storage_key, storage_value, storage_proof)
 }
+
+#[no_mangle]
+pub extern "C" fn ocall_sgx_init_quote(
+    ret_ti: *mut sgx_target_info_t,
+    ret_gid: *mut sgx_epid_group_id_t,
+) -> sgx_status_t {
+    ocall_bridge::ffi::init_quote::sgx_init_quote(ret_ti, ret_gid, OCallBridge::get_ra_api())
```

Comment:
Here, we will always know the concrete type of the `RemoteAttestationApi`. So we can make `sgx_init_quote` generic define the concrete type here.

Reply:
I think these lines of code in the main.rs are worth getting rid of `dyn`.

Reply:
The problem is, that we don't know the concrete types here, as soon as we extract the `OCallBridge` to a separate crate, that's the whole point of the separate crate. The concrete types get injected at runtime by the untrusted worker.